matter_server/server/server.py

```diff
@@ -106,7 +112,22 @@ async def start(self) -> None:
         await self.device_controller.start()
         await self.vendor_info.start()
         mount_websocket(self, "/ws")
-        self.app.router.add_route("GET", "/", self._handle_info)
+        self.app.router.add_route("GET", "/info", self._handle_info)
+
+        # Host dashboard if the prebuilt files are detected
+        if DASHBOARD_DIR_EXISTS:
+            self.logger.debug("Detected dashboard files on %s", str(DASHBOARD_DIR))
```

Comment:
We can store the string of the dashboard directory in a local variable since we need it multiple times.

Reply:
done